core/src/com/projectkorra/projectkorra/util/DamageHandler.java

```diff
@@ -158,6 +157,8 @@ public static void damageEntity(final Entity entity, Player source, double damag
 		if (ability == null) {
 			return;
 		}
+
+		if (entity.hasMetadata("BendingImmunity")) return;
```

Comment:
Is "`BendingImmunity`" meant to be temporary or permanent? If the latter, we should use persistent data instead.

Reply:
Temporary.